.lintstagedrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  '*.{ts,tsx,vue,css,less,scss,html,htm,md,markdown}': 'prettier --write',
```

Comment:
The project doesn't have Typescript files, are they included here to be future proof?

Reply:
Yes, these are the supported formats of Prettier so I just included all of them. It doesn't matter if some formats are not, yet, used in this repo.